infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -193,8 +193,8 @@ public SequencerServer(ServerContext serverContext) {
     * @param conflictParam The conflict parameter.
     * @return A conflict hash code.
     */
-    public String getConflictHashCode(UUID streamId, int conflictParam) {
-        return streamId.toString() + Integer.toString(conflictParam);
+    public String getConflictHashCode(UUID streamId, byte[] conflictParam) {
+        return streamId.toString() + Utils.bytesToHex(conflictParam);
```

Comment:
Because that involves generating a byte buffer and a byte array, but ok, if you insist...